### ISSUE TITLE
Unable to login/signin, without error log. ( both in my local install and sandbox server)

### ISSUE THREAD

**Issue by sg552** [tinode/chat#375]
### Unable to Login(Signin) 

I am not able to login neither in the sanbox server, nor my local server. 

username:  alice
password:  alice123

### Is this a bug report of a feature request?
- [x] Bug report

#### Server-side
- [x] sandbox url:  https://sandbox.tinode.co
- [x] Your own setup:
  * platform Ubuntu 18
  * TinodeWeb/0.16.4-rc3 (tinodejs/0.16.4-rc3)
  * database backend: MYSQL
  
#### Client-side
- [x] TinodeWeb/tinodejs: javascript client
  * Browser make and version. (TinodeWeb/0.16.4-rc3 (tinodejs/0.16.4-rc3))

### Steps to reproduce

1. open   https://sandbox.tinode.co
2. input username:  alice,  with password:   alice123
3. click the "signin" button

### Expected behaviour

user should be logged in successfully, and the chat window should come out. 

or should give message:  "password incorrect"

### Actual behaviour

web page hanged in "loading"

### Server-side log

I can't touch the offical sandbox server's log. 

In my local environment, the log looks like: 

```
2020/03/07 17:20:26 Listening for client HTTP connections on [:6060]
2020/03/07 17:20:35 ws: session started Jvv1pRGG-zw 1
2020/03/07 17:20:35 in: '{"hi":{"id":"65668","ver":"0.16.4-rc3","ua":"TinodeWeb/0.16.4-rc3 (Chrome/77.0; Linux x86_64); tinodejs/0.16.4-rc3","lang":"en-US"}}' ip='47.56.64.204:54402' sid='Jvv1pRGG-zw' uid=''
2020/03/07 17:20:37 ws: session started G2-goLWx-hI 2
2020/03/07 17:20:37 in: '{"hi":{"id":"129614","ver":"0.16.4-rc3","ua":"TinodeWeb/0.16.4-rc3 (Chrome/77.0; Linux x86_64); tinodejs/0.16.4-rc3","lang":"en-US"}}' ip='47.56.64.204:54404' sid='G2-goLWx-hI' uid=''
```

### Client-side log

in chrome developer tools console: 
```
[09:12:15:220] Connecting to:  wss://sandbox.tinode.co/v0/channels?apikey=AQEAAAABAAD_rAp4DJh05a1HAwFT3A6K
tinode.prod.js:1 [09:12:16:104] out: {"hi":{"id":"118024","ver":"0.16.4-rc3","ua":"TinodeWeb/0.16.4-rc3 (Chrome/77.0; Linux x86_64); tinodejs/0.16.4-rc3","lang":"en-US"}} 
tinode.prod.js:1 [09:12:16:387] in: {"ctrl":{"id":"118024","params":{"build":"rethinkdb:v0.16.4-rc3","maxFileUploadSize":8388608,"maxMessageSize":262144,"maxSubscriberCount":32,"maxTagCount":16,"ver":"0.16"},"code":201,"text":"created","ts":"2020-03-07T09:12:16.255Z"}}
```

and this is the log from chrome developer tools "network tag", I can see the websocket connection established: 

```
Request URL: wss://sandbox.tinode.co/v0/channels?apikey=AQEAAAABAAD_rAp4DJh05a1HAwFT3A6K
Request Method: GET
Status Code: 101 Switching Protocols
```

![Screenshot from 2020-03-07 17-22-43](https://user-images.githubusercontent.com/234533/76140724-54124380-6098-11ea-9b0d-f07c8d525b7a.png)

**Comment by sg552** [comment]
thanks so much!

**Comment by or-else** [comment]
Yes, it's a known bug. It's been fixed in devel https://github.com/tinode/webapp/commit/527a01bebb7a1cfe3e32a1098161481474b6357b but I have not updated the sandbox yet. Just press the Sign In button again.

**Comment by or-else** [comment]
I've released the v0.16.4-rc4 with the fix.

**Comment by sg552** [comment]
thanks so much!  it works well now!
